manifests/cf-manifest/operations.d/300-upgrade-cflinuxfs2-for-cve-mitigation.yml

```diff
@@ -0,0 +1,9 @@
+- path: /releases/name=cflinuxfs2/url
+  type: replace
+  value: https://storage.googleapis.com/cf-deployment-compiled-releases/cflinuxfs2-1.216.0-ubuntu-trusty-3586.16-20180612-151850-131406428.tgz
+- path: /releases/name=cflinuxfs2/version
+  type: replace
+  value: 1.216.0
+- path: /releases/name=cflinuxfs2/sha1
+  type: replace
+  value: 4ef1de398eec2f6bab1015a462c67fc43f6ba159
```

Comment:
A small style note, instead of 3 replace operations, we could do this with a single one (eg https://github.com/alphagov/paas-cf/commit/678dd66a)